pkg/systemd/overview-cards/lastLogin.jsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { InfoCircleIcon } from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+import { LoginMessage } from "../loginMessage.jsx";
```

Comment:
Feels a bit like over-splitting to me -- moving the loginMessage here, as helper function or full component, would keep things simpler and easier to read IMHO.

Reply:
I'll add it as a helper then, I was also a bit on the fence about leaving this separate.

Reply:
Moved into one file.